pkg/oauth/handlers/default_auth_handler.go

```diff
@@ -43,10 +43,8 @@ const (
 	// WarningHeaderOpenShiftSource is the name of the agent adding the warning header
 	WarningHeaderOpenShiftSource = "Origin"
 
-	warningHeaderCodeIndex  = 1
-	warningHeaderAgentIndex = 2
-	warningHeaderTextIndex  = 3
-	warningHeaderDateIndex  = 4
+	warningHeaderCodeIndex = 1
```

Comment:
eh... maybe keep the old indices in a comment?

Reply:
hm, dead code is bad code. I would rather suggest the person to go through the git commit history. WDYT?

Reply:
dead code indeed is bad, yet seeing indices that seem randomly chosen is puzzling and will raise more eyebrows. I'd rather we keep the unused consts commented :neutral_face:

Reply:
ok, I added a comment.

Reply:
```
// There were more indexes, but where unused.
```